static/app/components/stream/group.tsx

```diff
@@ -480,12 +495,16 @@ function BaseGroupRow({
 
       {withChart && !displayReprocessingLayout && issueTypeConfig.stats.enabled && (
         <ChartWrapper narrowGroups={narrowGroups}>
-          <GroupChart
-            stats={groupStats}
-            secondaryStats={groupSecondaryStats}
-            showSecondaryPoints={showSecondaryPoints}
-            showMarkLine
-          />
+          {organization.features.includes('issue-stream-new-events-graph') ? (
+            groupChart
```

Comment:
i'd define move the ternary here to avoid GroupStatusChart from being called just to be unused